dataset/tmp/0.9.0_exported_dbs/tinysnb/workAt.csv

```diff
@@ -0,0 +1,4 @@
+a.ID,b.ID,r.year,r.grading,r.rating
```

Comment:
All the above files should not be in the commit.

Reply:
Will think about how to best add the exported dbs to CI. Let's just get the infrastructure in for now.

A nice to have would be a Python or Bash script that we can run to automatically create exported dbs for all the test datasets for any given Kuzu version.